From patchwork Fri Aug  2 07:29:57 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Julien Thierry <julien.thierry.kdev@gmail.com>
X-Patchwork-Id: 11072745
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 70E0913AC
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri,  2 Aug 2019 07:30:06 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 4BF5328780
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri,  2 Aug 2019 07:30:06 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 3A736287C9; Fri,  2 Aug 2019 07:30:06 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No,
 score=-7.9 required=2.0 tests=BAYES_00,DKIM_ADSP_CUSTOM_MED,
	FREEMAIL_FROM,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id D322328780
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri,  2 Aug 2019 07:30:05 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S2389653AbfHBHaE (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Fri, 2 Aug 2019 03:30:04 -0400
Received: from foss.arm.com ([217.140.110.172]:45670 "EHLO foss.arm.com"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1730790AbfHBHaE (ORCPT <rfc822;kvm@vger.kernel.org>);
        Fri, 2 Aug 2019 03:30:04 -0400
Received: from usa-sjc-imap-foss1.foss.arm.com (unknown [10.121.207.14])
        by usa-sjc-mx-foss1.foss.arm.com (Postfix) with ESMTP id 31648337;
        Fri,  2 Aug 2019 00:30:04 -0700 (PDT)
Received: from e112298-lin.cambridge.arm.com (usa-sjc-imap-foss1.foss.arm.com
 [10.121.207.14])
        by usa-sjc-imap-foss1.foss.arm.com (Postfix) with ESMTPA id
 609153F575;
        Fri,  2 Aug 2019 00:32:11 -0700 (PDT)
From: Julien Thierry <julien.thierry.kdev@gmail.com>
To: kvm@vger.kernel.org
Cc: will@kernel.org, Julien Thierry <julien.thierry.kdev@gmail.com>
Subject: [PATCH kvmtool] README: Update my email address
Date: Fri,  2 Aug 2019 08:29:57 +0100
Message-Id: <1564730997-6693-1-git-send-email-julien.thierry.kdev@gmail.com>
X-Mailer: git-send-email 1.9.1
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

My @arm.com address is gonna stop working. Update README information
with an address people can use to actually reach me.

Signed-off-by: Julien Thierry <julien.thierry.kdev@gmail.com>
---
 README | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README b/README
index 591f84f..d3c2d3a 100644
--- a/README
+++ b/README
@@ -116,4 +116,4 @@ Maintainers
 -----------
 
 kvmtool is maintained by Will Deacon <will@kernel.org> and Julien Thierry
-<julien.thierry@arm.com>.
+<julien.thierry.kdev@gmail.com>.
